client/src/components/NavBarLogin.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import PropTypes from "prop-types";
+
+const NavBarLogin = props => {
+  const { account, classes, setLoggedOutAccount } = props;
+
+  const loginLink = (
+    <li className={classes.userLogin}>
+      <Link className={`${classes.link} ${classes.lastItem}`} to="/login">
+        Login
+      </Link>
+    </li>
+  );
+
+  const userGreeting = (
+    <li className={classes.userLogin}>
+      <h4>Hello, {`${account.firstName} ${account.lastName} `}</h4>
+    </li>
+  );
+
+  const logoutLink = (
+    <li>
+      <button
+        className={`link ${classes.lastItem}`}
+        onClick={setLoggedOutAccount}
+      >
+        Logout
+      </button>
+    </li>
+  );
+
+  return !account || !account.email ? (
+    loginLink
+  ) : (
+    <React.Fragment>
+      {account.firstName && userGreeting}
+      {account.email && logoutLink}
+    </React.Fragment>
```

Comment:
I'm not sure what the firstName and email have to do with userGreeting and logoutLink. They're possibly not required.
I made a modification to the userGreeting here that calls the modification above, if you decide to go that route.
```suggestion
      {getUserGreeting(account)}
      {logoutLink}
    </React.Fragment>
```